lib/next_ls/runtime.ex

```diff
@@ -248,4 +249,10 @@ defmodule NextLS.Runtime do
       true
     end
   end
+
+  defp dispatch_to_extensions(state, message) do
+    Registry.dispatch(state.registry, :extensions, fn entries ->
+      for {pid, _} <- entries, do: send(pid, message)
+    end)
+  end
```

Comment:
i prefer to not extract private functions for things that are only used once

Reply:
Oops, agreed -- I was using it twice because I thought it was needed elsewhere but forgot to un-extract it after 😊